From ce6681b615fd215f23be0d7a76f6c6120ba70221 Mon Sep 17 00:00:00 2001
From: Alberto Planas <aplanas@gmail.com>
Date: Thu, 28 Feb 2019 15:45:28 +0100
Subject: [PATCH] mount: fix extra -t parameter

If 'fstype' parameter is not set in Linux environments, salt will
build a mount command with an empty -t value, making the command
fail.
---
 salt/modules/mount.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/salt/modules/mount.py b/salt/modules/mount.py
index 97263252c7..5903dea151 100644
--- a/salt/modules/mount.py
+++ b/salt/modules/mount.py
@@ -1220,7 +1220,6 @@ def mount(name, device, mkmnt=False, fstype='', opts='defaults', user=None, util
             args += ' -v {0}'.format(fstype)
         else:
             args += ' -t {0}'.format(fstype)
-
     cmd = 'mount {0} {1} {2} '.format(args, device, name)
     out = __salt__['cmd.run_all'](cmd, runas=user, python_shell=False)
     if out['retcode']:
-- 
2.22.0


